_config.yml

```diff
@@ -17,7 +17,7 @@ title: "Galaxy Training!"
 email: your-email@domain.com
 description: A collection of tutorials generated and maintained by Galaxy Community Members across the world
 url: "https://galaxyproject.github.io/training-material/" # the base hostname & protocol for your site, e.g. http://example.com
-github_repository: "https://github.com/galaxyproject/training-material"
+github_repository: "https://github.com/galaxyproject/training-material/"
```

Comment:
By adding the leading slash, we generate `//` URLs, which are not very nice. It would be better to make sure that all links have the following form: `{{ site.url}}/foo`.